app/controllers/courses_controller.rb

```diff
@@ -4,7 +4,8 @@ class CoursesController < ApplicationController
   before_action :render_feedback_component, only: :show
 
   def show
-    @course = Course
+    @course = Rails.cache.fetch(course_cache_key, expires_in: 15.minutes) do
```

Comment:
This means that any changes to course details will take (at most) 15 minutes to update on Find if the course is in the cache. Is that ok? @davidgisbey @duncanjbrown